docs/configuration/index.md

```diff
@@ -1377,7 +1377,13 @@ These Historical configurations can be defined in the `historical/runtime.proper
 
 In `druid.segmentCache.locations`, *freeSpacePercent* was added because *maxSize* setting is only a theoretical limit and assumes that much space will always be available for storing segments. In case of any druid bug leading to unaccounted segment files left alone on disk or some other process writing stuff to disk, This check can start failing segment loading early before filling up the disk completely and leaving the host usable otherwise.
 
-In `druid.segmentCache.locationSelectorStrategy`, one of leastBytesUsed or roundRobin could be specified to represent the strategy to distribute segments across multiple segment cache locations. The leastBytesUsed which is the default strategy always selects a location which has least bytes used in absolute terms. The roundRobin strategy selects a location in a round robin fashion oblivious to the bytes used or the capacity. Note that `if druid.segmentCache.numLoadingThreads` > 1, multiple threads can download different segments at the same time. In this case, with the leastBytesUsed strategy, historicals may select a sub-optimal storage location because each decision is based on a snapshot of the storage location status of when a segment is requested to download.
+In `druid.segmentCache.locationSelectorStrategy`, one of leastBytesUsed, roundRobin, random, or mostAvailableSize could be specified to represent the strategy to distribute segments across multiple segment cache locations.
+The leastBytesUsed strategy, which is the default strategy, always selects a location which has least bytes used in absolute terms.
```

Comment:
Sorry, I should have been more clear. [My previous comment](https://github.com/apache/druid/pull/8879#discussion_r369299047) was about itemizing this paragraph so that readers can find the description of each strategy more easily. This change unfortunately doesn't make any difference in the [compiled doc](https://github.com/apache/druid/blob/25f0d4009dd9cbb59e428a0b447d252dcdc50df8/docs/configuration/index.md#storing-segments). I think this paragraph can be more organized with bullet items or a table. What do you think?